src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -67,6 +67,7 @@ def load_arguments(self, _):
         c.argument('max_count', type=int, validator=validate_nodes_count)
         c.argument('enable_vmss', action='store_true')
         c.argument('node_zones', zones_type, options_list='--node-zones', help='(PREVIEW) Space-separated list of availability zones where agent nodes will be placed.')
+        c.argument('enable_pod_security_policy', action='store_true', options_list=['--enable-pod-security-policy'])
```

Comment:
i believe `options_list=['--enable-pod-security-policy']` is redundant, the `options_list` is only usefully when you want to provide short ones